tests/jsx-split-attrs/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -122,8 +122,7 @@ long_open_long_children = (
         colour="blue"
         size="large"
         submitLabel="Sign in with Google"
-      />
-      d
+      />d
```

Comment:
If I write `<div with="long attributes..." />blah`, the "blah" should really be on its own line if the div breaks.